src/BuildScriptGeneratorCli/Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -19,6 +21,10 @@ public static IServiceCollection AddOptionsServices(this IServiceCollection serv
                 ServiceDescriptor.Transient<IConfigureOptions<BuildScriptGeneratorOptions>, BuildScriptGeneratorOptionsSetup>());
             services.TryAddEnumerable(
                 ServiceDescriptor.Transient<IConfigureOptions<NodeScriptGeneratorOptions>, NodeScriptGeneratorOptionsSetup>());
+            services.TryAddEnumerable(
+                ServiceDescriptor.Transient<IConfigureOptions<DotNetCoreScriptGeneratorOptions>, DotNetCoreScriptGeneratorOptionsSetup>());
+            services.TryAddEnumerable(
+                ServiceDescriptor.Transient<IConfigureOptions<PhpScriptGeneratorOptions>, PhpScriptGeneratorOptionsSetup>());
```

Comment:
nit: do we need to add python later? (not in this PR)

Reply:
Yes, I am adding it in another PR.